### PR TITLE
Chore: readme update proposal with code examples, custom marker/…

### PR DESCRIPTION
**Pull request by fuentesloic**
## Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Chore update

## Description
Few various proposals, feel free to discard some or all of them 😄 

ADDED Leaflet js library link https://github.com/Leaflet
ADDED detail on the new `npx` command
ADDED `LMarker` positioned at the center of the `LMap` example
ADDED `LIcon` example using custom icon marker
ADDED data manipulation with props
ADDED TS usage example

## Checklist:
- [ ] ~My change requires a change to the documentation.~ maybe the opposite ♾️ 
- [ ] ~I have updated the documentation accordingly.~ well, it's the whole purpose of my PR 🌸 
- [ ] ~I have added tests to cover my changes (if not applicable, please state why)~ only doc update 😇

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,11 +22,13 @@ This module is really just about making it work with Nuxt 3 without the need to
 
 ## Quick Setup
 
+Run `npx` command to add `nuxt3-leaflet` into your `package.json` and `nuxt.config.ts`
```
Comment (fuentesloic) [inline README.md:25]: @danielroe relatives to https://github.com/fuentesloic/nuxt-stripe/pull/28#discussion_r1560384387

**Comment by Gugustinette** [comment]
A few things are not working in your PR, a syntax problem on `onMounted` and some weird calls of `position`.

I believe it should be like this :

```vue
<template>
  <div style="height:100svh; width:100svw">
    <LMap
      ref="map"
      :zoom="zoom"
      :center="position().center"
    >
      <LTileLayer
        url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png"
        attribution="&amp;copy; <a href=&quot;https://www.openstreetmap.org/&quot;>OpenStreetMap</a> contributors"
        layer-type="base"
        name="OpenStreetMap"
      />
      <!-- Example with a marker in the center of the map -->
      <LMarker
          v-if="hasMarker"
          :lat-lng="position().latLng"
        >
          <!-- Example with a custom icon set in public folder  -->
          <LIcon
            :icon-size="[32, 37]"
            :icon-anchor="[16, 37]"
            icon-url="/marker.png"
          />
        </LMarker>
    </LMap>
  </div>
</template>

<script setup lang="ts">
// Use leaflet library types
import type { PointExpression, LatLngExpression } from "leaflet"

const zoom = ref(6)

// You could re-use your map component passing props!
defineProps<{
  hasMarker: boolean
}>()

onMounted(() => {
  // and ... you could create animation on load
  zoom.value = 12
})

const position = () => {
  const position = [47.21322, -1.559482]

  // Example matching position to LMap's center and LMarker's lat-lng types
  return {
    latLng: position as LatLngExpression,
    center: position as PointExpression,
  }
}
</script>

<style>
body {
  margin: 0;
}
</style>
```

Would be cool to check that everything works and update your pull request. 👍

**Comment by Gugustinette** [comment]
Also, you can push a new page in the playground with an exemple for the marker if you want.

**Comment by fuentesloic** [comment]
@Gugustinette I will try this week-end! 🚀 thanks for the proposal! 👍

**Pull request closed without merging**